v2/bolt_store.go

```diff
@@ -105,6 +105,10 @@ func (b *BoltStore) initialize() error {
 	return tx.Commit()
 }
 
+func (b *BoltStore) EmitStats() bbolt.Stats {
```

Comment:
Maybe just `Stats`?  "Emit" makes me think that it's sending them somewhere.

Reply:
Yeah that's fair

Reply:
Done